graphite.lua

```diff
@@ -219,50 +221,82 @@ local function collect_stats()
 	end
 end
 
+_M.stop = function()
+	if common_stat_fiber ~= nil then
+		fiber.kill(common_stat_fiber:id())
+		common_stat_fiber = nil
+	end
+
+	if stat_fiber ~= nil then
+		fiber.kill(stat_fiber:id())
+		stat_fiber = nil
+	end
+
+	if sock ~= nil then
+		sock:close()
+		sock = nil
+	end
+end
+
+_M.reset = function()
```

Comment:
The semantics (the fact that stop() is also called) of this function is unclear from its naming. I'd rather prefer to have an optional argument passed to stop() instead.